scripts/docker-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+pushd `dirname $0` > /dev/null
+ST2DOCSROOT="${PWD%/*}"
+popd > /dev/null
+
+docker build -t st2/st2docs -f $ST2DOCSROOT/Dockerfile $ST2DOCSROOT
```

Comment:
I afraid it gets `meta`: I use `make` from docker...  So will keep it a bit separate for now.

While doing it I saw that so many things can be off-loaded from Makefile if docker was the way of building... But taking it simple for now.